src/lib/profanity.js

```diff
@@ -0,0 +1,5 @@
+export function isProfane(messageText) {
+  // TODO(lperson) future: return different categories, such as profane, abusive, etc.
```

Comment:
Drive-by inline review from an old friend. Check out https://github.com/web-mech/badwords for a customizable (with not great, but acceptable, defaults) JS-based profanity filter.

For more categories, the [Toxic Comment Challenge](https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/overview) resulted in models that could categorize online text into toxic, severely toxic, obscene, threatening, insulting, and identity-hating categories. If we end up caring a lot about differentiating between modalities of profanity, I can look into synthesizing one of these models into a regex-based routine.